content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -997,11 +997,12 @@ See also:
 | `--create-only`   | No       | Creates a new migration based on the changes in the schema but does not apply that migration. Run `migrate dev` to apply migration. |         |
 | `--skip-seed`     | No       | Skip triggering seed                                                                                                                |         |
 | `--skip-generate` | No       | Skip triggering generators (for example, Prisma Client)                                                                             |         |
-| `--help` / `--h`  | No       | Displays the help message                                                                                                           |
+| `--name` / `-n`   | No       | Name the migration (e.g. `prisma migrate dev --name added_job_title`)                                                               |         |
+| `--help` / `-h`   | No       | Displays the help message                                                                                                           |
 
 <Admonition type="info">
 
-If a [schema drift](/concepts/components/prisma-migrate/shadow-database#detecting-schema-drift) is detected while running `prisma migrate dev` using `--create-only`, you will be prompted to reset your database.
+If a [schema drift](/concepts/components/prisma-migrate/shadow-database#detecting-schema-drift) is detected while running `prisma migrate dev` using `--create-only`, you will be prompted to reset your database. If you choose to reset your database, the `--create-only` flag will be ignored and the migration will be applied. If a migration name is not provided, `prisma migrate dev` will prompt you for a name.
```

Comment:
> If a migration name is not provided, `prisma migrate dev` will prompt you for a name.

Is that related to the schema drift detection in any way, or is this just unrelated information?